en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-260-to-300.md

```diff
@@ -1635,6 +1635,14 @@ Follow the instructions below to move all the existing API Manager configuration
         ```
 
 4.  Copy the keystores (i.e., `client-truststore.jks`, `wso2cabon.jks` and any other custom JKS) used in the previous version and replace the existing keystores in the `<API-M_3.0.0_HOME>/repository/resources/security` directory.
+    
+    **Note:** From API Manager 3.0.0 onwards, keystores could be configured as follows.
+    
+    ```toml
+    - keystore.primary - the default keystore. If only primary is configured, the primary keystore will be used for internal and SSL related encryption and decryption tasks. 
```

Comment:
- the default keystore. >>>Shall we change "the" to "The"
- If only primary is configured, >>> Isn't it better to mention as to what in particular you are referring to when you say "primary" to give it more context.